task_01/src/szyszki/task1.java

```diff
@@ -18,16 +27,17 @@ public PanWyjatek(String error)
 	public static void main(String[] args) throws PanWyjatek
 {
 		//
-		PanWyjatek mrException = new PanWyjatek("Something went wrong :(");
+		
 		int a, b, c;
 		try 
 		{
 			a = Integer.parseInt(args[0]);
 			b = Integer.parseInt(args[1]);
 			c = Integer.parseInt(args[2]);
 		}
-		catch (PanWyjatek p)
+		catch (Exception p)
 		{
+			PanWyjatek mrException = new PanWyjatek("Something went wrong :(");

```

Comment:
Dasz radę zawrzeć to w jednej linijce? :)

Reply:
widzę tu tylko jedną linijkę, o co chodzi? xd